tests/p2p/test_discovery.py

```diff
@@ -196,6 +196,6 @@ def random_address():
 
 
 def random_node():
-    seed = force_bytes("".join(random.sample(string.ascii_lowercase, 10)))
+    seed = to_bytes(text="".join(random.sample(string.ascii_lowercase, 10)))
```

Comment:
this seems overly complicated.  Maybe change this to just `seed = os.urandom(10)` to seed it with 10 random bytes?

Reply:
Do we know that the seed can contain any possible bytes? Maybe there is a reason we need or want it to be valid ascii lowercase letter bytes only.

Reply:
Gonna leave this as-is for now and merge.